phoenix-core/src/main/java/org/apache/phoenix/compile/ScanRanges.java

```diff
@@ -694,4 +694,138 @@ public TimeRange getRowTimestampRange() {
         return rowTimestampRange;
     }
 
+    public List<KeyRange> getRowKeyRanges(){
+
```

Comment:
Do you want to keep this blank line? I don't see we have such blank line at the beginning of function body in other functions.

Reply:
Ok